### ISSUE TITLE
Layout/MultilineMethodCallIndentation false positive in 1.84.1

### ISSUE THREAD

**Issue by seandilda** [rubocop/rubocop#14833]
In 1.84.1, correctly formatted code is flagged as an error by `Layout/MultilineMethodCallIndentation`

--------

## Steps to reproduce the problem

```ruby
# frozen_string_literal: true

method(key: value.chain1.chain2
                 .chain3)

```

## Expected behavior

I would expect no error.  This is using the default values for `Layout/MultilineMethodCallIndentation` of aligned.  Since `.chain3` is aligned with the dot of the first method in the chain, I would expect this to be acceptable.

## Actual behavior

```
$ rubocop --debug test.rb
For /app: configuration from /app/.rubocop.yml
Default configuration from /home/docker-user/bundle/ruby/3.3.0/gems/rubocop-1.84.1/config/default.yml
Plugin configuration from /home/docker-user/bundle/ruby/3.3.0/gems/rubocop-rails-2.34.3/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /app/test.rb
Loading cache from /home/docker-user/.cache/rubocop_cache/f949e70474bdfbf9be0eb032b218ce426938c3fb/8c3e361d53661224764d286241e06b0a70c26148/97d8e6c9212104493d0a3800574218ee21eefcff
C

Offenses:

test.rb:4:18: C: [Correctable] Layout/MultilineMethodCallIndentation: Align .chain3 with value.chain1.chain2 on line 3.
                 .chain3)
                 ^^^^^^^

1 file inspected, 1 offense detected, 1 offense autocorrectable
Finished in 0.20254784999997355 seconds
```

## RuboCop version

```
$ rubocop -V
1.84.1 (using Parser 3.3.10.1, Prism 1.9.0, rubocop-ast 1.49.0, analyzing as Ruby 3.3, running on ruby 3.3.10) [x86_64-linux]
  - rubocop-rails 2.34.3

```
